script.module.trakt/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.trakt"
        name="trakt.py"
-       version="3.0.0"
+       version="3.1.0"
        provider-name="Fuzeman, Razzeee">
```

Comment:
correct author is `Dean Gardiner`, https://github.com/fuzeman/trakt.py/blob/3.1.0/setup.cfg